Marlin/ultralcd.cpp

```diff
@@ -125,7 +125,8 @@ uint16_t max_display_update_time = 0;
 
   #if ENABLED(AUTO_BED_LEVELING_UBL)
     extern bool ubl_has_control_of_lcd_panel;
-    extern uint8_t ubl_encoderDiff;
+    volatile uint8_t ubl_encoderDiff = 0; // Volatile because it's changed by Temperature ISR button update
+    bool ubl_has_control_of_lcd_panel = false;
```

Comment:
A `bool` plus `extern bool` for  `ubl_has_control_of_lcd_panel` here probably breaks something. Anyway, we should be getting rid of `ubl_has_control_of_lcd_panel` soon.